tests/test_snql_api.py

```diff
@@ -575,6 +575,30 @@ def test_suspect_spans_data(self) -> None:
         assert len(data) == 1
         assert data[0]["array_spans_exclusive_time"] > 0
 
+    def test_app_start_type(self) -> None:
```

Comment:
@evanh, I'm unsure if this is the best way to test this new column. Can you maybe guide me in the right direction, please?

Reply:
This looks good to me!